### DISCUSSION TITLE
How to get notified for "highResourceUsage"

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5770] [FAQs] [answered]
I've noticed the Ant Media Server sends out "highResourceUsage" warning calls when the server is overloaded due to excessive CPU usage.
I'd like to learn more about how to effectively handle these warning messages.

1. How can we configure the threshold value for the "highResourceUsage" warning? Is it possible to customize it based on our server's specific requirements?
2. Are there best practices for setting up alerts via webhook when such warnings are received?

Thank you

**Comment by lastpeony** [comment, marked as answer]
You can configure threshold values for cpu and minimum ram usage. The default CPU limit is 75%, and the minimum free RAM is 10 MB.

Locate `red5.properties` file in `usr/local/antmedia/conf` directory.

For cpu usage percentage thresholds, add/change `server.cpu_limit=75`

For minimum ram usage threshold in MB, add/change `server.min_free_ram=10`

If your CPU load percentage exceeds the defined threshold, and the available free RAM falls below the specified minimum, it will trigger a high resource usage event. If you have configured a webhook URL in your server settings, the server will send a POST webhook containing resource usage details to the specified endpoint.

To configure the server hook URL, add the below line to the `red5.properties` file. Please remember that this webhook end point is different from the application webhook url.

`server.statusWebHookURL={your webhook url}`

After making the above changes, restart the Ant Media Server.
